docs/System-Requirements.md

```diff
@@ -1,10 +1,18 @@
 # Hardware and Software Requirements
 
 This page describes various system software required to run the cBioPortal.
+Note that one can also run cBioPortal using [Docker](Docker-Prerequisites.md),
+which is considerably easier.
 
 ## Hardware
 
-Hardware requirements will vary depending on the volume of users you anticipate will access your cBioPortal instance.  As a guideline, we run [cbioportal.org](https://www.cbioportal.org) on an Intel Xeon 2.60GHz/8-core processor with 192GB of RAM. Minimally, 2GB of RAM is needed to run a cBioPortal instance. The public database consumes ~50 GB of disk space. If you do not plan to import public studies, depending on the size of your private data, 10GB of disk space may be sufficient.
+Hardware requirements will vary depending on the volume of users you anticipate
+will access your cBioPortal instance.  As a guideline, we run
+[cbioportal.org](https://www.cbioportal.org) on an AWS r5.xlarge instance with
+32 GB and 4 vCPUs. Minimally, 2GB of RAM is needed to run a cBioPortal
+instance. The public database consumes ~50 GB of disk space. If you do not plan
+to import public studies, depending on the size of your private data, 10GB of
+disk space may be sufficient.
 
```

Comment:
@khzhu sorry, could you clarify? The system requirements should be the same on-premise? I can detail the spec of the AWS machine some more, but 32GB of RAM and 4 cores should suffice to run public portal. This is a better indication than we had with our old on-premise machine, cause it ran many instances of cBioPortal and therefore needed more resources (in the docs it mentions 192GB RAM)

Reply:
@inodb , sorry for the confusion. It will be nice, if you can summarize in a table structure, something like:

> 
| Platform  | CPUs  | RAM |  Storage |
| ------------- | ------------- | ------------- | ------------- |
| on-premise  | 4  | 32  |  50 |
| aws  | 4 | 32  | 50  |

Reply:
put this in a new section `Hardware-Requirements`